catalog/transformer/catalog-transformer-pdf/src/test/groovy/ddf/catalog/transformer/input/pdf/PdfInputTransformerSpecTest.groovy

```diff
@@ -15,8 +15,11 @@ package ddf.catalog.transformer.input.pdf
 
 import ddf.catalog.content.operation.ContentMetadataExtractor
 import ddf.catalog.data.Metacard
+import ddf.catalog.data.MetacardType
 import ddf.catalog.data.impl.AttributeDescriptorImpl
 import ddf.catalog.data.impl.BasicTypes
+import ddf.catalog.data.impl.MetacardTypeImpl
+import ddf.catalog.data.impl.types.*
```

Comment:
It shouldn't have if you've installed our IJ code style. It defaults to 99 imports from a package before collapsing to `*`.